docs/overview/what-is-waku.md

```diff
@@ -0,0 +1,56 @@
+---
+title: What is Waku?
+slug: /
+---
+
+Waku is a family of peer-to-peer protocols that provide **secure and private communication** in a decentralized environment. It is designed to operate in **resource-limited environments**, making it suitable for various decentralized applications (dApps).
```

Comment:
The phrasing might make a developer think that Waku is **only** for resource-limited environments. Maybe worth noting it can also be used as a node or desktop application.